docs/guide/snaps.md

```diff
@@ -76,7 +76,9 @@ Once you've connected to Flask, smash that `Send Hello` button, and watch the sn
 
 ## Where to Go from Here
 
-- If you're interested in developing your own snap, your next destination is probably [the snaps development guide](./snaps-development-guide.html).
-- You may also wish to check out the first functional key management snap ever made, [the Filecoin Snap, or `filsnap`](https://github.com/Chainsafe/filsnap/).
-- Check out the list of [Awesome MetaMask Snaps](https://github.com/piotr-roslaniec/awesome-metamask-snaps) and other resources.
-- Currently, snaps are mostly discovered through word-of-mouth. As the system matures, MetaMask will provide means for both developers and users to discover snaps.
+- If you're interested in developing your own snap, your next destination is probably [the Snaps development guide](./snaps-development-guide.html).
+- You can also try the following example snaps. Each one is fully functional and open-source:  
+  - [Filsnap for Filecoin support](https://github.com/Chainsafe/filsnap/)
+  - [BitcoinSnap for Bitcoin support](https://github.com/KeystoneHQ/btcsnap)
+  - [Password Manager Snap](https://github.com/ritave/snap-passwordManager)
+- If you have questions or need help, join our [discussion board](https://github.com/MetaMask/snaps-skunkworks/discussions).
```

Comment:
Should we link the Discord too?

Reply:
No, I really don't want to encourage that. I will continue to support it for the people who are using it, but officially I will nudge people toward our GitHub Discussion Board since that is a much better place to ask for help.